### PR TITLE
Removed some sections from Automated Upgrades topic

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1565344

Preview Builds: 
http://file.rdu.redhat.com/~ahardin/04192018/web-console-in-automated/upgrading/automated_upgrades.html

http://file.rdu.redhat.com/~ahardin/04192018/web-console-in-automated/upgrading/manual_upgrades.html

### CODE REVIEW

File: upgrading/automated_upgrades.adoc [hunk 1]
```diff
@@ -519,10 +518,10 @@ on what additional features you have previously enabled.
 |Feature |Next Step
 
 |Aggregated Logging
-|xref:automated-upgrading-efk-logging-stack[Upgrade the EFK logging stack.]
+|xref:manual_upgrades.adoc#manual-upgrading-efk-logging-stack[Upgrade the EFK logging stack.]
 
 | Cluster Metrics
-| xref:automated-upgrading-cluster-metrics[Upgrade cluster metrics.]
+xref:../../manual_upgrades.adoc#manual-upgrading-cluster-metrics[Upgrade cluster metrics.]
```
Comment (juzhao) [inline upgrading/automated_upgrades.adoc]: Cluster Metrics

Should show Aggregated Logging and Cluster Metrics  in the table, but Cluster Metrics is missing in this PR
******************************************************************************************************************
Feature                      Next Step    
Aggregated Logging | Upgrade the EFK logging stack.
Cluster Metrics         | Upgrade cluster metrics.
******************************************************************************************************************

**Comment by ahardin-rh** [comment]
The | was missing. This is now fixed. Thanks!

File: upgrading/automated_upgrades.adoc [hunk 2]
```diff
@@ -188,7 +188,6 @@ otherwise it is left as is. For either scenario, masters are automatically
 labeled with the `master` node role, and non-master, non-infrastructure nodes
 are labeled with the `compute` node role.
 +
-ifdef::openshift-enterprise[]
```
Comment (vikram-redhat) [inline upgrading/automated_upgrades.adoc]: @ahardin-rh - this removal of ifdef wasn't followed up with an endif removal. Consequently, there is an extra endif in this file. Can you check and fix?

Comment (ahardin-rh) [reply upgrading/automated_upgrades.adoc]: @vikram-redhat Yes, fixed in https://github.com/openshift/openshift-docs/pull/9119

**Pull request merged**